perfkitbenchmarker/scripts/object_storage_api_tests.py

```diff
@@ -1200,13 +1237,49 @@ def ListConsistencyBenchmark(storage_schema, host_to_connect=None):
                final_result)
 
   # Final clean up: delete the objects still remaining.
-  DeleteObjectsConcurrently(storage_schema,
+  DeleteObjectsConcurrently(backend,
                             per_thread_objects_written,
-                            host_to_connect,
                             per_thread_objects_deleted)
   return final_result
 
 
+def TestBackend(backend):
```

Comment:
I changed to `ValidateProvider`, since we are now calling backends providers. I don't think it should be part of the provider class hierarchy, because it's not needed to use a provider. But it's also not part of the benchmarker, so I moved it to its own script inside the `script-tests/integration` directory. How does that sound?